src/CMakeLists.txt

```diff
@@ -837,17 +875,15 @@ install(DIRECTORY "${src_numpy_core}/lib/"
 install(DIRECTORY "${src_numpy_core}/include/"
         DESTINATION include)
 
-install(FILES ${CAPNP_LINK_LIBRARIES}
-        DESTINATION lib)
 foreach(directory ${CAPNP_INCLUDE_DIRS})
   install(DIRECTORY "${directory}/capnp"
           DESTINATION include)
   install(DIRECTORY "${directory}/kj"
           DESTINATION include)
 endforeach()
 
-install(FILES ${src_lib_static_nupiccore_combined}
-        DESTINATION lib)
+#install(FILES ${src_lib_static_nupiccore_combined}
```

Comment:
@scottpurdy, thanks for catching it - forgot to delete. This was another benefit of the new function `MERGE_STATIC_LIBRARIES`. Since we can now treat src_lib_static_nupiccore_combined like any other library target, it is now installed simply by adding it to the `install(TARGETS ...)` function above.